active/social-media.md

```diff
@@ -0,0 +1,80 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships and campaigns.
```

Comment:
Not sure if generating revenue is one of the goals of this WG.

Reply:
Re-phrased to:
> Protect the framework's long-term viability through campaigns and sponsor relations.

I think this working group can and should help with supporting the success of funding or sponsor campaigns but yes, is not resposible for generating revenue. Let me know if you have a nice way to say this 😁

Reply:
I think the new working works well. It’s very common for non-profits to fundraise on social media these days, I imagine that’d work great for Django too.